docs/changes/newsfragments/258.doc

```diff
@@ -0,0 +1 @@
+Fixed a typo in cross validation by 'Dante Culaciati'
```

Comment:
Can you replace `'Dante Culaciati'` with ``` `Dante Culaciati`_```

Reply:
Ok, will do so now.